moped-etl/README.md

```diff
@@ -1 +1,34 @@
+
+If you are working in a shared environment, protect the file to being read-only by your user account. This can be accomplished using the `chmod 600 <filename>` command in your terminal. 
+
+#### 2) Build the image
+Execute the following to build the ETL image: `docker build -t atd-moped-etl --no-cache .` 
+
+If you are developing the image itself, the `--no-cache` argument can be removed. It is useful when you may have old intermediate layers to your image, that may contain manifests of software available apk, dpkg, yum or other package management systems.
+
+#### 3) Development of ETL scripts
+You can start the container using the following command for local development.
+
+`docker run -it --rm -p 5555:5555 -v $(pwd)/app:/app -v $(pwd)/data:/data --env-file local_moped_dev_knack.env --env HASURA_ENDPOINT=http://$(./get_ip.sh):8080/v1/graphql atd-moped-etl bash`
+
+There is an important difference between local development of Moped ETL scripts and how they will run in production. In production, the graphql-engine endpoint will be known and will have a fixed IP or hostname. It will be defined in the environment file along with all the other specifics about the deployment. 
+
+Local development has a complication however. It is likely that you will be self hosting the graphql-engine endpoint on your local machine. Outside of the container, you likely are referring to that endpoint using the `localhost` hostname. However, inside the container itself, `localhost` points to the local network interface of the container, which is separate and apart from the host's `localhost.` Because the container does not have knowledge of the network configuration of the host, the IP where the graphql-endpoint can be found must be explicitly supplied to the container, again through the use of environment variables. A bash script is provided, `get_ip.sh`, which will return the IP assigned to the active network interface or the interface currently holding the broadcast flag. To confirm that this is returning the IP you want to pass to the container as part of the graphql-engine endpoint, execute the script and substitute the IP into the URL seen in the above command. This URL should return a JSON object, likely with an error stating "resource does not exist." While it does indicate an error, that error pertains to the malformed query sent to the endpoint, and the fact that an error was returned at all indicates the graphql-engine endpoint is available on that network IP. 
```

Comment:
Sorry but this is simply untrue. There are actually a couple of methods to make containers connect to a running cluster (i.e., a create-and-append a localized docker network by name, using `--network host` with `localhost`, or use `host.docker.internal`, to name a few). 

Here, I was able to ping Hasura without any IP configuration:
 
<img width="1096" alt="2022-01-04_08-12-21" src="https://user-images.githubusercontent.com/5282430/148072064-be44f97b-7b31-4506-8fd4-1c3e8e932a35.png">

<img width="1237" alt="2022-01-04_08-13-11" src="https://user-images.githubusercontent.com/5282430/148072092-62f453ad-95ba-4a62-b4cc-32dc2f78a611.png">

I'd say get rid of `get_ip.sh` and simplify the run process with one of the two options above, it would be easier and more stable to run on other platforms.

Reply:
Thanks for your suggestion. I don't want to use `--network host` because it's not easy to predict that all the ports that the container wants to expose are open to be bound to, but your suggestion for `host.docker.internal` is exactly what this needs. I'll amend the PR to utilize it and deprecate the `get_ip.sh`.

Reply:
I will wait to hear from them before I make any changes.

Reply:
I have updated the ETL system to utilize the `host.docker.internal` hostname.

I will await feedback from John and Mateo per your request -- @johnclary , I've added you to the list of reviewers if that's helpful. Thank you both!

Reply:
TIL. Looks good to me!